site_tmpl/emails/email_ccadd.html

```diff
@@ -32,9 +32,27 @@
                 <p>Submit your crew chief report <a href="{% get_base_url %}{% url 'my:report' ccinstance.event.id %}">here</a>.</p>
 
                 <p>Submit crew hours <a href="{% get_base_url %}{% url 'my:hours-list' ccinstance.event.id %}">here</a>.</p>
-
+		{% if is_new_cc %}
+			<p>By the way, congratulations on taking your first event as
+			Crew Chief! Since this is your first time as a CC, here
+			are some helpful resources. 
+			<ul>
+					<li>Tech Bible -
+						<a
+								href="https://lnl.wpi.edu/bible">lnl.wpi.edu/bible</a></li>
+					<li>Technical Director -
+						<a href="mailto:lnl-td@wpi.edu">lnl-td@wpi.edu</a></li>
+					<li>Vice President - lnl-vp@wpi.edu</li>
```

Comment:
If we create a link for the Technical Director's email, we should do the same for the other ones:
```suggestion
					<li>Vice President - <a href="mailto:lnl-vp@wpi.edu">lnl-vp@wpi.edu</a></li>
```